app/Http/Controllers/SanatizerController.php

```diff
@@ -1,139 +1,25 @@
 <?php
 
 namespace App\Http\Controllers;
+use App\Sanatizer;
 
 use Illuminate\Support\Facades\File;
 
 class SanatizerController extends Controller
 {
+    public $rawHTMLFolderPath;
+    public $cleanedSurasPath;
 
     public function __construct()
     {
         $this->rawHTMLFolderPath = storage_path('nonSanatizedSuras');
         $this->cleanedSurasPath = storage_path('SanatizedSuras');
-        
     }
 
-    public function sanatize()
+    public function runSanatization()
```

Comment:
since the name of the class `SanatizerController`, we can have the name of this method (function) simply as `run`... it would make sense `SanatizerController#run` is actually running the sanatization